### ISSUE TITLE
Automatic Region Switching

### ISSUE THREAD

**Issue by Crypta-Eve** [Crypta-Eve/provi-i#4]
When a player is detected as changing region then have the ability to follow the player into the new region.

**Comment by Crypta-Eve** [comment]
Would also benefit from #3

**Comment by AndrisArickson** [comment]
Toggle on and off. Sometimes you want to watch the intel of a nearby region while you are in another region and automatic switching would defeat this.

**Comment by Crypta-Eve** [comment]
Agreed, will most definetly be configurable. At this stage my thoughts are 3 options. 1. Normal (No Switch) 2. Prompted (each time a change is prompt the user to change region) 3. Auto (Change without prompting)

**Comment by raykai** [comment]
would love this !
